mpmath/function_docs.py

```diff
@@ -1872,7 +1872,7 @@
 
 .. math ::
 
-    \eta(s) = \sum_{k=0}^{\infty} \frac{(-1)^k}{k^s}
+    \eta(s) = \sum_{k=1}^{\infty} \frac{(-1)^{k-1}}{k^s}
```

Comment:
Lower limit seems to be correct, according to the A-A, eq. 23.2.19 or https://mathworld.wolfram.com/DirichletEtaFunction.html